### DISCUSSION TITLE
SteelSeries GG App Causing Flicker

### DISCUSSION THREAD

**Discussion by TwistedBlizzard** [tidal-music/discussions#17] [Report bug]
The SteelSeries GG app titled "TIDAL - for OLED Displays" causes flickering and unresponsiveness on SteelSeries Apex Pro keyboards.

**Comment by stefanmoro** [comment]
Hey @TwistedBlizzard, and welcome to the forum.

This GitHub discussions forum aims at handling technical discussions around the TIDAL Developer Platform (TIDAL API, open source projects etc).

As this question relates to an existing partner integration, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com).

Thanks

**Reply by TwistedBlizzard** [reply]
Ah, no worries! Wasn't sure if it was the same team handling the plugin. Cheers!
